packages/client/src/runtime/RequestHandler.ts

```diff
@@ -30,11 +31,10 @@ export type RequestParams = {
   clientMethod: string
   callsite?: CallSite
   rejectOnNotFound?: RejectOnNotFound
-  runInTransaction?: boolean
```

Comment:
In all instances it was set, it was set to `Boolean(transactionId)` so this looks redundant to me. I refactored all the usages so they look at `transaction` field instead.